### ISSUE TITLE
updateRadioGroupButtons inside callModule does not update input values

### ISSUE THREAD

**Issue by statnmap** [dreamRs/shinyWidgets#89]
HI,
I try to use `updateRadioGroupButtons` inside a shiny module. This updates correctly the visual aspect of the radio Buttons but not the returned value. If you use the same exact code using classical `radioButtons`, this works correctly.  
I tested the dev version.  
Below a reproducible example.

```r
# devtools::install_github("dreamRs/shinyWidgets")

mod_testUI <- function(id) {
  ns <- NS(id)
  tabItem(
    tabName = "test",
    box(
      radioGroupButtons(
        # radioButtons(
        inputId = ns("bouton"),
        label = "Choisir :",
        choices = c("2007", "2008"),
        selected = "2007"
      ),
      column(12, style = "font-weight:bold;color:red;",
             textOutput(ns("out_text")))
    ))
}

mod_test <- function(input, output, session) {
  updateRadioGroupButtons(
    # updateRadioButtons(
    session, "bouton",
    choices = c("A", "B", "ok"),
    selected = "ok"# ,
    # inline = TRUE
  )

  output$out_text <- renderText({
    paste(input$bouton)
  })

}

library(shiny)
library(shinyWidgets)
if (interactive()) {
  ui <- fluidPage(
    mod_testUI("plop")
  )

  server <- function(input, output, session) {
    callModule(mod_test, "plop")
  }

  shinyApp(ui, server)
}
```

**Comment by pvictor** [comment]
Thanks for the example!
Same issue than Pierre, it's fixed (version 0.4.3.920).

Victor

**Comment by statnmap** [comment]
Merci. Ça marche !
